exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -119,10 +120,41 @@ func timeSeriesSignature(metric *otlp.Metric, labels *[]prompb.Label) string {
 	return b.String()
 }
 
+// convertResourceAttributes to labels takes a metric's respective ResourceMetric to extract resource attributes and convert to a
+// array of StringKeyValue
+func convertResourceAttributesToLabels(resource *resource.Resource) ([]*common.StringKeyValue, error) {
+	attrs := resource.Attributes
+
+	// convert attributes to strings since labels are only strings. return error if unsupported types exist
+	// Assumption here is that no metrics should have Array or Kvlist resource attribute values, hence only an error is returned if one is found
+	newLabels := make([]*common.StringKeyValue, len(attrs))
+	for index, attr := range attrs {
+		newLabel := common.StringKeyValue{Key: attr.GetKey()}
+		switch attr.GetValue().GetValue().(type) {
+		case *common.AnyValue_StringValue:
+			newLabel.Value = attr.GetValue().GetStringValue()
+		case *common.AnyValue_BoolValue:
+			newLabel.Value = strconv.FormatBool(attr.GetValue().GetBoolValue())
+		case *common.AnyValue_IntValue:
+			newLabel.Value = strconv.FormatInt(attr.GetValue().GetIntValue(), 10)
+		case *common.AnyValue_DoubleValue:
+			// option 'G' formats floats with an exponent (eg. d.dddde±dd) for sufficiently large values
+			newLabel.Value = strconv.FormatFloat(attr.GetValue().GetDoubleValue(), 'G', -1, 64)
```

Comment:
Just checking, is this conventional or used in other places in the collector?

Reply:
This was a way that I found myself, I will look through the collector to see if another method exists and is used

Reply:
![image](https://user-images.githubusercontent.com/24356083/97380341-2e42e480-189d-11eb-9814-ccba3290bf84.png)

As we can see in this picture this method is indeed used quite a bit within the collector, but difference being that the format string used is 'f', which never adds an exponent, hence I will update the code to reflect this